gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/AttachmentHandlerService.java

```diff
@@ -19,9 +19,7 @@
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import uk.nhs.adaptors.connector.model.PatientAttachmentLog;
-import uk.nhs.adaptors.pss.translator.config.SupportedFileTypes;
 import uk.nhs.adaptors.pss.translator.exception.InlineAttachmentProcessingException;
-import uk.nhs.adaptors.pss.translator.exception.UnsupportedFileTypeException;
```

Comment:
Can we remove the `UnsupportedFileTypeException` class as well?

Reply:
I think so, I will delete it.